### PR TITLE
load-balanced-instances module

### PR DESCRIPTION
**Pull request by RLuckom**
Module that creates load-balanced group of instances with DNS records

### CODE REVIEW

File: terraform-modules/load-balanced-instances/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+`service`: the name of the service
+
+`dns_project`: the name of the project in which the DNS will be deployed
+
+`dns_region`: The Google region in which to deploy the DNS records e.g. `us-central1`
+
+`google_compute_ssl_certificate_red`: the name of the red ssl cert as stored in google
+
+`google_compute_ssl_certificate_black`: the name of the black ssl cert as stoblack in google
```
Comment (kcibul) [inline terraform-modules/load-balanced-instances/README.md]: stoblack -> stored?

Comment (RLuckom) [reply terraform-modules/load-balanced-instances/README.md]: ah the dangers of find/replace. thanks

File: terraform-modules/load-balanced-instances/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+`dns_zone_name`: The name of the DNS zone according to Google, e.g. `dsde-perf-broad`
+
+`owner`: the owner of the deployment
+
+`service`: the name of the service
+
+`dns_project`: the name of the project in which the DNS will be deployed
+
+`dns_region`: The Google region in which to deploy the DNS records e.g. `us-central1`
```
[the remark this answers is not in the document]
Comment (RLuckom) [reply terraform-modules/load-balanced-instances/README.md]: Good catch, I think this was a holdover from when I needed to actually define a DNS provider block--region is required there. But with the provider passed in I don't think this is required.

File: terraform-modules/load-balanced-instances/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+This module creates a cluster of identical GCE instances for deploying a containerized service,
+puts them behind a HTTP load balancer, and assigns DNS records for each instance and the
+load balancer. It also creates a GCS bucket for storing configs for the service and gives access
+to the GCS bucket to the specified service account.
+
```
[the remark this answers is not in the document]
Comment (RLuckom) [reply terraform-modules/load-balanced-instances/README.md]: good catch, added

File: terraform-modules/load-balanced-instances/variables.tf [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  name = "${var.google_compute_ssl_certificate_red}"
+}
+data "google_compute_ssl_certificate" "terra-env-wildcard-ssl-certificate-black" {
+  name = "${var.google_compute_ssl_certificate_black}"
+}
+# Network
+variable "google_network_name" {
+}
+
+# Thurloe SA
```
[the remark this answers is not in the document]
Comment (RLuckom) [reply terraform-modules/load-balanced-instances/variables.tf]: good catch, fixed

File: terraform-modules/load-balanced-instances/variables.tf [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+variable "config_reader_service_account" {
+}
+data "google_service_account" "config_reader" {
+  account_id = "${var.config_reader_service_account}"
+}
+
+#
+# Thurloe - Common Vars
```
[the remark this answers is not in the document]
Comment (RLuckom) [reply terraform-modules/load-balanced-instances/variables.tf]: done

**Pull request merged**